packages/react-component-tree/src/load-child-component.js

```diff
@@ -0,0 +1,10 @@
+var React = require('react'),
+    loadChild = require('./load-child.js');
+
+class LoadChildComponent extends React.Component {
+  loadChild(childName, a, b, c, d, e, f) {
```

Comment:
This might actually be more efficient, stole it from the React source code a while ago. But I'm going to ES6 the s*it out of this repo before merging. Wanted to make sure everything works first before I "celebrate". :D